Source/Scene/Camera.js

```diff
@@ -2745,6 +2723,7 @@ define([
         Cartesian3.clone(camera.up, result.up);
         Cartesian3.clone(camera.right, result.right);
         Matrix4.clone(camera._transform, result.transform);
+        camera.frustum.clone(result.frustum);
```

Comment:
You should check what type of frustum the result has. By default, the camera creates a `PerspectiveFrustum`, but it could be an `OrthographicFrustum`. In practice, it'll never be a `PerspectiveOffCenterFrustum`, but that's on the road map.